be/src/storage/rowset/bitmap_index_writer.cpp

```diff
@@ -40,14 +40,64 @@ namespace starrocks {
 
 namespace {
 
+static const size_t roaring_get_size_fastpath_threshold = 1024;
```

Comment:
1. The global variable should starts with `k_`. such as `k_roaring_get_size_fastpath_threshold`
2. It is not neccessary to make this variable a global variable, make it a class variable is enough.